index.html

```diff
@@ -88,7 +88,9 @@
                         }
                     ],
                     // maximum entries to retain in conductor history
-                    records: 10
+                    records: 10,
+                    // maximum duration between start and end bounds
```

Comment:
lets add a clear comment that this limit is in milliseconds

Reply:
done